streaming-binlog/src/main/resources/debezium_config.json

```diff
@@ -0,0 +1,18 @@
+{
+   "debeziumConfigurations":  {
+      "databaseName" : "mysql",
+      "databaseHostName" : "localhost",
+      "databaseServerId" : "77",
+      "databasePort" : "3306",
+      "databaseUser" : "root",
+      "databasePassword" : "debezium",
+      "databaseServerName" : "mysql",
+      "databaseSchema" : "openmrs",
+      "offsetStorage" : "org.apache.kafka.connect.storage.FileOffsetBackingStore",
+      "databaseOffsetStorage" : "data/offset.dat",
+      "databaseHistory" : "data/dbhistory.dat",
+      "snapshotMode" : "initial",
+      "fhirDebeziumEventConfigPath" : "./utils/dbz_event_to_fhir_config.json",
```

Comment:
Where is this path used? And why is it not in `DebeziumConfiguration`?

Reply:
going to remove it